packages/webapp/components/invite/Referral.tsx

```diff
@@ -0,0 +1,221 @@
+import useMedia from '@dailydotdev/shared/src/hooks/useMedia';
+import { Modal } from '@dailydotdev/shared/src/components/modals/common/Modal';
+import {
+  ModalKind,
+  ModalSize,
+} from '@dailydotdev/shared/src/components/modals/common/types';
+import CloseButton from '@dailydotdev/shared/src/components/CloseButton';
+import { JoinPageProps } from './common';
+
+export function Referral({
+  referringUser,
+  redirectTo,
+  token,
+}: JoinPageProps): ReactElement {
+  const router = useRouter();
+  const genericReferral = useFeature(feature.genericReferral);
+  const { completeAction } = useActions();
+  const { trackEvent } = useAnalyticsContext();
+  const { displayToast } = useToastNotification();
+  const { user, refetchBoot, showLogin } = useAuthContext();
+  const isMobile = !useMedia([mobileL.replace('@media ', '')], [true], false);
+  const isLaptopL = useMedia([laptopL.replace('@media ', '')], [true], false);
+  const {
+    mutateAsync: onAcceptMutation,
+    isLoading,
+    isSuccess,
+  } = useMutation(acceptFeatureInvitation, {
+    onSuccess: async () => {
+      await Promise.all([
+        completeAction(ActionType.AcceptedGenericReferral),
+        refetchBoot(),
+      ]);
+      router.push(redirectTo);
+    },
+    onError: (err: ApiErrorResult) => {
+      const message = err?.response?.errors?.[0]?.message;
+      displayToast(message ?? DEFAULT_ERROR);
+    },
+  });
+  const [isVideoOpen, setVideoOpen] = useState(false);
+
+  const onVideoModalClose = () => {
+    setVideoOpen(false);
+  };
+
+  const handleAcceptClick = () => {
+    const handleAccept = () =>
+      onAcceptMutation({
+        token,
+        referrerId: referringUser.id,
+        feature: ReferralCampaignKey.Generic,
+      });
+
+    if (!user) {
+      return showLogin({
+        trigger: AuthTriggers.GenericReferral,
+        options: {
+          onLoginSuccess: handleAccept,
+          onRegistrationSuccess: handleAccept,
+        },
+      });
+    }
+
+    // since in the page view, query params are tracked automatically,
+    // we don't need to send the params here explicitly
+    trackEvent({ event_name: AnalyticsEvent.AcceptGeneralReferralInvitation });
+
+    return handleAccept();
+  };
+
+  useEffect(() => {
+    if (genericReferral === GenericReferral.Control) {
+      return;
+    }
+
+    router.push(redirectTo);
+    // router is an unstable dependency
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [redirectTo, genericReferral]);
+
+  return (
+    <div
+      style={{
+        background: `url(${cloudinary.referralCampaign.genericReferral.backgroundDark})`,
+        backgroundSize: 'cover',
+      }}
+      className="flex overflow-hidden relative flex-col laptop:flex-row flex-auto items-center p-2 laptop:p-6 h-[100vh]"
+    >
+      <span className="absolute top-8 left-1/2 laptop:left-12 -translate-x-1/2 laptop:translate-x-0 scale-[1.375]">
```

Comment:
Why the scale? did we not have the icon already in the right size?
It should not differ from the other invite pages I don't think.

Reply:
> Why the scale? did we not have the icon already in the right size?

I could not find any other one except of what was already in the join page for AISearchInvite

> It should not differ from the other invite pages I don't think.

Did doubt that as well, though the name in the DR referral program & Ux Improvements made me think maybe this could be any of the changes/improvements. 

Checking with design now in [thread](https://dailydotdev.slack.com/archives/C05P9ET7S9K/p1699012788634189?thread_ts=1699011522.350609&cid=C05P9ET7S9K), definitely best to not add custom stuff.

Reply:
Removed the scale, the icon is same across the app